renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -1218,6 +1218,70 @@ void APIENTRY _glClearBufferData(GLenum target, GLenum internalformat, GLenum fo
 
 #pragma endregion
 
+#pragma region GLES Compatibility
+
+void APIENTRY _glGetTexImage(GLenum target, GLint level, GLenum format, GLenum type, void *pixels)
+{
+  if((format == eGL_DEPTH_COMPONENT && !HasExt[NV_read_depth]) ||
+     (format == eGL_STENCIL && !HasExt[NV_read_stencil]) ||
+     (format == eGL_DEPTH_STENCIL && !HasExt[NV_read_depth_stencil]))
+  {
+    // return silently, check was made during startup
+    return;
+  }
+
+  GLint width = 0, height = 0, depth = 0;
+  internalGL->glGetTexLevelParameteriv(target, level, eGL_TEXTURE_WIDTH, &width);
+  internalGL->glGetTexLevelParameteriv(target, level, eGL_TEXTURE_HEIGHT, &height);
+  internalGL->glGetTexLevelParameteriv(target, level, eGL_TEXTURE_DEPTH, &depth);
+
+  GLint texture = 0;
+  internalGL->glGetIntegerv(TextureBinding(target), (GLint *)&texture);
+
+  GLenum attachment = eGL_COLOR_ATTACHMENT0;
+  if(format == eGL_DEPTH_COMPONENT)
+    attachment = eGL_DEPTH_ATTACHMENT;
+  else if(format == eGL_STENCIL)
+    attachment = eGL_STENCIL_ATTACHMENT;
+  else if(format == eGL_DEPTH_STENCIL)
+    attachment = eGL_DEPTH_STENCIL_ATTACHMENT;
+
+  GLuint fbo = 0;
+  internalGL->glGenFramebuffers(1, &fbo);
+
+  PushPopFramebuffer(eGL_FRAMEBUFFER, fbo);
+
+  size_t sliceSize = GetByteSize(width, height, 1, format, type);
+  GLenum binding = TextureTarget(target);
```

Comment:
Yes, this was because of cube faces. I fixed it and also I added some comments for unhandled texture types. However for 1d and 1d array types I think it's quite useless since GLES does not supports these.